src/qiboml/models/keras.py

```diff
@@ -0,0 +1,65 @@
+"""Keras interface to qiboml layers"""
+
+from dataclasses import dataclass
+
+import keras
```

Comment:
You're using Keras, but that's undeclared as a dependency in `pyproject.toml`.

Reply:
I double-checked this and it seems that `keras` gets installed through the `tensorflow` dependency, do you think it is better to have it explicitely declared anyway?